ddp-workspace/projects/ddp-dsm-ui/src/app/participant-page/participant-page.component.ts

```diff
@@ -545,6 +553,39 @@ export class ParticipantPageComponent implements OnInit, OnDestroy, AfterViewChe
     }
   }
 
+  kitValueChanged( value: any, parameterName: string, sample: Sample ): void {
+    let v;
```

Comment:
I would rather call variables more self-explanatory names, for example instead of "v", maybe it would be better to call it a "currentValue".